docs/src/routes/docs/[...1]overview/[...2]onboard.js-migration-guide.md

```diff
@@ -46,7 +46,7 @@ const onboard = Onboard({
 })
 ```
 
-You can find the full list of initialization options in our docs here.
+You can find the full list of initialization options in our docs [here](https://onboard.blocknative.com/docs/packages/core#options).
```

Comment:
Should this be a relative link or absolute?

Reply:
As is will work great! 🤓

Reply:
We should use relative links whenever possible. You can read about them in the kit docs documentation here.

https://kit-docs.svelteness.dev/docs/markdown/extensions#links

Reply:
@gesquinca sounds good. All the docs currently use absolute so I can try and go in and fix those as well.